setup/setup

```diff
@@ -12,3 +12,6 @@ cd "capnproto-c++-$VERSION" || exit 1
 make -j6 check
 sudo make install
 cd ../
+export DATABASE_DIR="/var/db/scratchql"
+sudo mkdir $DATABASE_DIR
+sudo chown -R $eval$(whoami):$eval$(whoami) $DATABASE_DIR
```

Comment:
do you really need the eval here?

Reply:
It seems like it:
% sudo rm -rf /var/db/scratchql; export DATABASE_DIR='/var/db/scratchql'; sudo mkdir $DATABASE_DIR; sudo chown -R $whoami:$whoami $DATABASE_DIR
 % ls -lh /var/db                                                                                                       CODE/ScratchQL (travis-fs-fix ⚡) TemplarAE
total 20K
-rw-r--r-- 1 root root 5.3K May 24 16:12 Makefile
drwxr-xr-x 2 root root 4.0K Mar 12 21:08 nscd
drwxr-xr-x 2 root root 4.0K Jun 18 18:27 scratchql
drwx--x--x 3 root root 4.0K May 11 13:18 sudo
------------

Reply:
with eval you get:
`drwxr-xr-x 2 felix felix 4.0K Jun 18 18:32 scratchql`

Reply:
the first result is because you execute `whoami` as root - of course, it results in root:root

I'm actually surprised that the second one works :D

Reply:
I assume $eval gets executed by you before the whole command gets executed by root. Funny, and elegant solution (I mean, elegant for a *workaround*

Reply:
well, I'll have a hard time to run chown as non-root user on a root-owned directory

Reply:
try harder ^^
no, you're right, you need root there.
I meant that, the whole setup thingy, is a workaround in the end

Reply:
just imagine the distant future, in which we do this with a proper package manager and a proper configuration setup XD